.coveragerc

```diff
@@ -1351,6 +1351,16 @@ omit =
     homeassistant/components/upnp/__init__.py
     homeassistant/components/upnp/device.py
     homeassistant/components/upnp/sensor.py
+    homeassistant/components/upnp/*
```

Comment:
These lines have nothing to do with your PR and should not be added, right?

Reply:
uh yeah... made a mistake during the merge. fixed it now - thanks!